README.md

```diff
@@ -63,22 +63,124 @@ hello-world:
 | Key | Requirement | Description |
 |:---:|:---:|:---|
 | `metadata` | **Mandatory** | General description of the app (see below). |
-| `git_url` | **Mandatory** | Link to the source code git repository. |
+| `releases` | **Mandatory** | List the source(s) for releases, e.g., a git repository. |
 | `categories` | Optional | If provided, must be one of the valid categories specified in [`categories.yaml`](https://github.com/aiidalab/aiidalab-registry/blob/main/categories.yaml). |
 
-
 ### Valid keys for app metadata:
 
 | Key | Requirement | Description |
 |:---:|:---:|:---|
 | `title` | **Mandatory** | The title will be displayed in the list of apps in the application manager. |
 | `description` | **Mandatory** | The description will be displayed on the detail page of your app. |
 | `authors` | Optional | Comma-separated list of authors. |
-| `logo` | Optional | Url to a logo file (png or jpg). |
 | `state` | Optional | One of<br>- `registered`: lowest level - app may not yet be in a working state. Use this to secure a specific name.<br>- `development`: app is under active development, expect the occasional bug.<br>- `stable`: app can be used in production. |
 | `documentation_url` | Optional | The link to the online documentation of the app (e.g. on [Read The Docs](https://readthedocs.org/)). |
 | `external_url` | Optional | General homepage for your app. |
 
+#### Release specification
+
+Releases are specified in the form of a list, where each list entry corresponds to one or more tagged commits of a git repository branch.
+In case that it corresponds to multiple commits, the release entry is called a *release line*.
+
+This is how we can specify a release pointing to a specific version:
+```yaml
+hello-world:
+  releases:
+  - git+https://github.com/aiidalab/aiidalab-hello-world.git@v1.0.0
+```
+The value behind the `@` symbol must in this case point to a specific tag.
+Note that the above example is short-hand for:
+```yaml
+hello-world:
+  releases:
+  - url: git+https://github.com/aiidalab/aiidalab-hello-world.git@v1.0.0
+```
+Specifying a release entry as dictionary allows us to override the environment specification (see next section) as well as the version name.
+
+On the other hand, if we want to add all tagged commits on a specific branch to the release line, we point to the branch name followed by a colon (`:`):
+```yaml
+hello-world:
+  releases:
+  - "git+https://github.com/aiidalab/aiidalab-hello-world.git@master:"
```

Comment:
Is this `:` also part of the `git+https` way of installing stuff? Or is this only used here?

(just asking)